src/ColumnSchema.php

```diff
@@ -54,4 +55,18 @@ public function dbTypecast(mixed $value): mixed
 
         return parent::dbTypecast($value);
     }
+
+    public function phpTypecast(mixed $value): mixed
+    {
+        if (is_string($value) && $this->getType() === SchemaInterface::TYPE_TIME) {
+            $value = explode(' ', $value)[1] ?? $value;
+        }
+
+        return parent::phpTypecast($value);
+    }
+
+    public function hasTimezone(): bool
```

Comment:
Need have a tests for public functions

Reply:
`ColumnSchema::hasTimezone()` can be replaced with two new abstract types `SchemaInterface::TYPE_TIMESTAMPTZ` and `SchemaInterface::TYPE_TIMETZ`

Tests will be added after review this option in `yiisoft/db` [#736](https://github.com/yiisoft/db/pull/736)